src/main/java/github/grace000/erversay/RouteHandlers/OptionsTwo.java

```diff
@@ -3,12 +3,13 @@
 import github.grace000.erversay.Request;
 import github.grace000.erversay.ResponseBuilder;
 
-import java.util.Arrays;
 
 import static github.grace000.erversay.Constants.Headers.OPTIONS_TWO_HEADER;
 import static github.grace000.erversay.Constants.StatusCodes.NOT_FOUND_STATUS;
 
 public class OptionsTwo implements RouteHandler {
+    private ResponseBuilder responseBuilder = new ResponseBuilder();
```

Comment:
Any reason you're creating this object here, instead of in the `handle` function?